app/src/main/res/drawable/messages_background_image.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<bitmap xmlns:android="http://schemas.android.com/apk/res/android"
+    android:src="@drawable/shapes_outlines"
+    android:tileMode="repeat"
+    android:tint="#D9D1CA">
```

Comment:
extract color string to values/colors.
It's easy, move cursor to it and alt+return